VERSION

```diff
@@ -1,2 +1,2 @@
 VERSION=1.0
-RELEASE=29
+RELEASE=32
```

Comment:
Q1: should we bump it up to 33?
Q2: are we maintaining/using it?

Reply:
No, we can keep it as 32 only for now. We can be bump it up during the next release.